build/avalanchego-apis/admin-api-c-chain.md

```diff
@@ -0,0 +1,186 @@
+# Admin API (C-Chain)
+
+This API can be used for debugging. Note that the Admin API is disabled by default. To run a node with the Admin API enabled, use [command line argument](../references/command-line-interface.md#c-chain-config) `--coreth-admin-api-enabled=true`.
+
+## Format
+
+This API uses the `json 2.0` RPC format.
+
+{% page-ref page="issuing-api-calls.md" %}
+
+## Endpoint
+
+```text
+/ext/bc/C/admin
+```
+
+## API Methods
+
+### admin.setLogLevel
+
+Sets the log level of the C-Chain.
+
+#### **Signature**
+
+```text
+admin.setLogLevel({level:string}) -> {success:bool}
+```
+
+* `level` is the log level to be set.
```

Comment:
What are the log levels they can provide?

Reply:
its already listed here https://docs.avax.network/build/references/command-line-interface#c-chain-config

![Screenshot 2021-11-01 at 18 58 07](https://user-images.githubusercontent.com/26006996/139718076-8655c02c-26d2-4693-9860-b72da91ce147.png)

should I list them again ?